### ISSUE TITLE
Prevent saving of the same animal to local storage

### ISSUE THREAD

**Issue by ryanpjhickey** [ryanpjhickey/Paws-to-Home#24]
We need to create an if statement to check if the current animal already exists in local storage. If it exists, do not save the animal to local storage again.

**Comment by abhijithp05** [comment]
@ryanpjhickey We need to check for the current animal in existing values of localStorage right?

**Comment by ryanpjhickey** [comment]
> @ryanpjhickey We need to check for the current animal in existing values of localStorage right?

Yes that is the idea. I would try to implement this myself but I am working on other projects right now so feel free to give it a go if you want to! Just put in a PR if you are able to get it working :)

**Comment by abhijithp05** [comment]
@ryanpjhickey Can you share some test cases so that I can run to understand the flow

**Comment by ryanpjhickey** [comment]
@abhijithp05 Sure. Here are two test cases I've created. Let me know if you need further clarification.

Test Case 1:

Description: User can save an animal to local storage.

Test Step: Have user save an animal by clicking "Add PetName to favorites."

Expected Result: Animal is saved to local storage and displayed to favorites.html

Status: Pass or Fail



Test Case 2:

Description: User cannot save two instances of the same animal to local storage.

Test Step: Have user save an animal by clicking "Add PetName to favorites." Refresh the page, search the same zip code again. Have user attempt to save animal from Test Case 1 again by clicking "Add PetName to favorites."

Expected Result: Animal is not saved to local storage again. Animal is still displayed on favorites.html, but only a singular time.

Status: Pass or Fail
